src/java.base/share/classes/java/lang/StringCoding.java

```diff
@@ -26,8 +26,15 @@
 
 package java.lang;
 
+import jdk.internal.util.Preconditions;
 import jdk.internal.vm.annotation.IntrinsicCandidate;
 
+import java.util.function.BiFunction;
+
+import static java.util.Objects.requireNonNull;
+import static jdk.internal.util.Preconditions.AIOOBE_FORMATTER;
+import static jdk.internal.util.Preconditions.checkFromIndexSize;
```

Comment:
Avoid import static of methods, it makes the code harder to read and understand where the methods come from.
Just import Preconditions.

Reply:
Fixed in 2672f7c1.